test/multi-core-indexer.test.js

```diff
@@ -366,6 +366,42 @@ test('Entries are re-indexed if index storage unlinked', async () => {
   await indexer2.close()
 })
 
+test('Entries can be explicitly reindexed with a startup option', async (t) => {
+  const cores = await createMultiple(3)
+  const [core1, core2, core3] = cores
+  const expectedIn1And2 = await generateFixtures([core1, core2], 3)
+  const expectedIn3 = await generateFixtures([core3], 3)
+
+  const storage = ram.reusable()
+
+  const indexer1 = new MultiCoreIndexer(cores, {
+    batch: async () => {},
```

Comment:
It would be good to capture these entries and verify that they are being indexed the first time.

Reply:
Done in 705ee8d3a54c1507c503b448f34be4c3e0c1b333.